wasi/wasi.go

```diff
@@ -358,8 +358,8 @@ func (w *WASIEnvironment) args_sizes_get(ctx *wasm.HostFunctionCallContext, args
 //     The total number of bytes written there is the value that args_sizes_get returns in argsBufSizePtr. The caller must ensure that
 //     the buffer has the enough size.
 //     Each *C.char pointer that can be obtained from argsPtr points to the beginning of each of these null-terminated strings.
-// See: https://github.com/WebAssembly/WASI/blob/snapshot-01/phases/snapshot/docs.md#-args_getargv-pointerpointeru8-argv_buf-pointeru8---errno
-// See: https://en.wikipedia.org/wiki/Null-terminated_string
```

Comment:
I noticed I suggested something not conventional before as we always didn't use a colon!